inc/Ui/PageDiff.php

```diff
@@ -191,33 +173,50 @@ protected function preProcess()
      */
     public function show()
     {
-        if (!isset($this->oldRevInfo, $this->newRevInfo)) {
+        if (!isset($this->Rev1, $this->Rev2)) {
             // retrieve form parameters: rev, rev2, difftype
             $this->handle();
             // prepare revision info of comparison pair, except PageConfrict or PageDraft
             $this->preProcess();
         }
 
+        // revision information object
+        [$Rev1, $Rev2] = [$this->Rev1, $this->Rev2];
+
+        // revision title
+        $rev1Title = trim($Rev1->showRevisionTitle() .' '. $Rev1->showCurrentIndicator());
+        $rev1Supple = ($Rev1->val('date'))
```

Comment:
"Supple" is an unclear shortcut, after seen the previous code I guess it means supplement?

Reply:
`$rev1Supple` holds change summary and editor, so "Supple" has changed to "Summary" 3712ae9